File: ScpProject/StrengthConApp/src/app/Models/SetsAndReps/Set.ts
export class Set {
    public Id: number;
    public Position: number;//the position in the order of sets
    public Percent: number;
    public Sets: number;
    public Reps: number;
    public Weight: number;
    public Minutes:number;
    public Seconds: number;
    public Distance: number;
    public RepsAchieved : boolean;
    public Other: string;
    public ParentWeekId: number = 0;
}

File: ScpProject/BL/BusinessObjects/SetsAndReps/Set.cs

namespace BL.BusinessObjects.SetsAndReps
{
    public class Set
    {
        public int Id { get; set; }
        public int Position { get; set; }
        public int? Sets { get; set; }
        public int? Reps { get; set; }
        public double? Percent { get; set; }
        public double? Weight { get; set; }
        public int ParentWeekId { get; set; }
        public int? Seconds { get; set; }
        public int? Minutes { get; set; }
        public string Distance { get; set; }
        public bool? RepsAchieved { get; set; }
        public string Other { get; set; }
    }
}
